chain/chain/src/flat_storage_creator.rs

```diff
@@ -402,9 +402,10 @@ impl FlatStorageShardCreator {
                         );
                         store_update.commit()?;
                         info!(target: "chain", %shard_id, %flat_head, %height, "Garbage collected {gc_count} deltas");
-                        if let Some(manager) = self.runtime.get_flat_storage_manager() {
```

Comment:
note: no meaningful change other than stop checking return value of get_flat_storage_manager()